compiler-cli/src/dependencies.rs

```diff
@@ -24,6 +24,84 @@ use crate::{
     http::HttpClient,
 };
 
+pub fn list() -> Result<()> {
+    let config = crate::config::root_config()?;
+    let manifest = read_manifest_from_disc()?;
```

Comment:
I chose to use this function, but realized there's also [`get_manifest`](https://github.com/gleam-lang/gleam/blob/235eb17ff473cef461cdca0478302e23ba610aea/compiler-cli/src/dependencies.rs#L318-L342). If I understand correctly, `read_manifest_from_disc` only reads the manifest from the toml file, whereas `get_manifest` reads and resolves the manifest versions. I wasn't sure which would be desired for this command, so went with the simpler of the two to start with. Which (or neither 🤷 😄) do you think would be best?

Reply:
I think if we are to read it from the disc we would want to create a friendly error message for the situation in which the manifest is absent, so the user knows how to process.

Having said that, is there any scenario in which the user would want to list the deps but not permit them to be resolved beforehand? I think it's likely OK to use the other function so that thet can be resolved and this command can succeed if run before dep resolution.

Reply:
Ah, yes! That makes sense to me. I hadn't considered what would happen if the user didn't have a manifest yet. I'll update things to use the other function 😄 Thank you!